### PR TITLE
bugfix optiongroupserializer missing

### PR DESCRIPTION
**Pull request by svandeneertwegh**
Added:
`option_group = AttributeOptionGroupSerializer()`

to:

```class OptionSerializer(OscarHyperlinkedModelSerializer):```

in file: oscarapi/serializers/product.py

### CODE REVIEW

**Comment by svandeneertwegh** [comment]
This fixes the problem mentioned at #324 !

File: oscarapi/serializers/product.py [hunk 1]
```diff
@@ -186,6 +186,7 @@ class Meta:
 
 
 class OptionSerializer(OscarHyperlinkedModelSerializer):
+    option_group = AttributeOptionGroupSerializer()
```
Comment (specialunderwear) [inline oscarapi/serializers/product.py:189]: Hi, I think this should be optional, the tests are failing atm!

**Review by specialunderwear — CHANGES_REQUESTED** [review]

File: oscarapi/serializers/product.py (hunk 1 above)
Comment (specialunderwear) [reply oscarapi/serializers/product.py:189]: Also I think it is important to demonstrate the problem in a test, to make sure there are no regressions in the future

**Comment by svandeneertwegh** [comment]
@specialunderwear when this is not applied i get errors on admin endpoints when i set:
`OSCARAPI_BLOCK_ADMIN_API_ACCESS = False`

File: Makefile [hunk 2]
```diff
@@ -22,7 +22,7 @@ coverage:
 	coverage run sandbox/manage.py test oscarapi --settings=sandbox.settings.block_admin_api_true
 	coverage run sandbox/manage.py test oscarapi --settings=sandbox.settings.block_admin_api_false
 	coverage report -m
-	coverage xml -i
+        coverage xml -i
```
Comment (specialunderwear) [inline Makefile:25]: This has to be a tab, you broke the tests by making them spaces

**Review by specialunderwear — CHANGES_REQUESTED** [review]

**Review by specialunderwear — APPROVED** [review]

**Pull request merged**